lib/inspec/rspec_json_formatter.rb

```diff
@@ -63,7 +63,7 @@ def stop(notification)
   private
 
   def format_example(example)
-    if example.metadata[:description_args].length > 0 && !example.metadata[:skip].nil?
+    if !example.metadata[:description_args].empty? && example.metadata[:skip]
```

Comment:
The first statement was a suggestion by Rubocop. The second is the same way of expressing the same thing.